schemas/gitaVerse.js

```diff
@@ -36,6 +36,23 @@ export default {
               }
             }
           },  
+          {
+            name: "tags",
```

Comment:
I don't know if we should have tags.  It just complicates things. Not many people use it. Instead we have full text search on algolia which will do a better job. What do you think? I might be wrong .

Reply:
sure we will remove the tags and try to make the commentary part with refrences to adapt multiple languages in commentaries too , let's see how it goes